tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -309,6 +309,7 @@
   "dbm_propagation_mode": "dbm_propagation_mode",
   "trace.remove_root_span_laravel_queue": "trace_remove_root_span_laravel_queue_enabled",
   "trace.remove_autoinstrumentation_orphans": "trace_remove_auto_instrumentation_orphans_enabled",
+  "DD_DISCOVERY_SERVICE_ENABLED": "discovery_service_enabled",
```

Comment:
Do we need to add the new config to [[config_norm_rules.json in dd-go](https://github.com/DataDog/dd-go/blob/2f7f1b882d0b77295a6d8b3e2f86ee9d11f5eb52/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json)](https://github.com/DataDog/dd-go/blob/2f7f1b882d0b77295a6d8b3e2f86ee9d11f5eb52/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json)?

BTW, I noticed a config with a similar name - `agent_discovery_enabled`. I’m not sure we use it or what it was originally meant for. Could it be intended for something similar to what you’re doing?